vaccine_feed_ingest/runners/in/arcgis/normalize.py

```diff
@@ -0,0 +1,174 @@
+
+    potentials = {
+        "Pfizer": schema.Vaccine(vaccine="pfizer"),
+        "Moderna": schema.Vaccine(vaccine="moderna"),
+        "Johnson & Johnson": schema.Vaccine(vaccine="janssen"),
+        "FRPP": schema.Vaccine(vaccine="frpp"),
+    }
+
+    inventory = []
+
+    for vf in vaccines_field:
+        try:
+            inventory.append(potentials[vf])
+        except KeyError as e:
+            logger.error("Unexpected vaccine type: %s", e)
+
+    if len(inventory) > 0:
+        return inventory
+
+    return None
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+    if site["attributes"]["Site_Phone"]:
+        sourcePhone = re.sub("[^0-9]", "", site["attributes"]["Site_Phone"])
+        if len(sourcePhone) == 11:
+            sourcePhone = sourcePhone[1:]
+        phone = f"({sourcePhone[0:3]}) {sourcePhone[3:6]}-{sourcePhone[6:]}"
+        contacts.append(schema.Contact(phone=phone))
+
+    if site["attributes"]["Site_Zotec_Link"]:
+        contacts.append(schema.Contact(website=site["attributes"]["Site_Zotec_Link"]))
+
+    if site["attributes"]["Site_Location_Info"]:
+        contacts.append(schema.Contact(other=site["attributes"]["Site_Location_Info"]))
+
+    if len(contacts) > 0:
+        return contacts
+
+    return None
+
+
+def _get_notes(site: dict) -> Optional[List[str]]:
+    if site["attributes"]["Site_Special_Inst"]:
+        return [site["attributes"]["Site_Special_Inst"]]
+
+    return None
+
+
+def _get_city(site: dict) -> Optional[str]:
+    address = site["attributes"]["Site_Address"]
+
+    return re.search("^.*[.,]\\s(.+), IN (?:[0-9]|-)+$", address).group(0)
+
+
+def _get_zip(site: dict) -> Optional[str]:
+    address = site["attributes"]["Site_Address"]
+
+    return re.search("^.*, IN ((?:[0-9]|-)+)$", address).group(0)
+
+
+def _get_normalized_location(site: dict, timestamp: str) -> schema.NormalizedLocation:
+    return schema.NormalizedLocation(
+        id=_get_id(site),
+        name=site["attributes"]["Name"],
+        address=schema.Address(
+            street1=site["attributes"]["Site_Address"],
+            street2=None,
+            city=_get_city(site),
+            state="IN",
+            zip=_get_zip(site),
+        ),
+        location=schema.LatLng(
+            latitude=site["geometry"]["y"],
+            longitude=site["geometry"]["x"],
+        ),
+        contact=_get_contacts(site),
+        languages=None,
+        opening_dates=None,
+        opening_hours=None,
+        availability=None,
+        inventory=_get_inventory(site),
+        access=None,
+        parent_organization=None,
+        links=None,
+        notes=_get_notes(site),
+        active=None,
+        source=schema.Source(
+            source="arcgis",
+            id=site["attributes"]["GlobalID"],
+            fetched_from_uri="https://services1.arcgis.com/WzFsmainVTuD5KML/ArcGIS/rest/services/COVID19_Vaccine_Site_Survey_API/FeatureServer/0",  # noqa: E501
```

Comment:
Please set this to `https://experience.arcgis.com/experience/24159814f1dd4f69b6c22e7e87bca65b`